### PR TITLE
[ZEPPELIN-1323] Add contribution guide for Zeppelin documentation

### PR DESCRIPTION
**Pull request by AhyoungRyu**
### What is this PR for?
Recently I added auto generated Table of Contents by #1031 and searching feature by #1266. So I wrote some guide to `docs/CONTRIBUTING.md` about how to use and how they can be applied to docs page. Hope this helps to new Zeppelin (docs) contributors :)

### What type of PR is it?
Documentation

### Todos
* [x] - Update `docs/README.md`

### What is the Jira issue?
[ZEPPELIN-1323](https://issues.apache.org/jira/browse/ZEPPELIN-1323)

### How should this be tested?
Just click `View` button on the top of the file diff :)

### Questions:
* Does the licenses files need update? no
* Is there breaking changes for older versions? no
* Does this needs documentation? no

### CODE REVIEW

**Comment by Leemoonsoo** [comment]
CI failure looks unrelated to this change.

Looks great to me!

File: docs/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+
+Default settings for TOC are definded in [here](https://github.com/apache/zeppelin/blob/master/docs/assets/themes/zeppelin/js/toc.js#L4).
+
+
+## Adding new pages
+If you're going to create new pages, there are some spots you need to add the location of the page.
+
+ - **Dropdown menu in navbar**: add your docs location to [_navigation.html](https://github.com/apache/zeppelin/blob/master/docs/_includes/themes/zeppelin/_navigation.html)
+ - **Main index**: add your docs below [What is the next?](http://zeppelin.apache.org/docs/0.7.0-SNAPSHOT/#what-is-the-next) section in [index.md](http://zeppelin.apache.org/docs/0.7.0-SNAPSHOT/#what-is-the-next) with short description. No need to do this if the page is for **Interpreters**.
```
Comment (felixcheung) [inline docs/CONTRIBUTING.md]: should this point to http://zeppelin.apache.org/docs/latest?

Comment (AhyoungRyu) [inline docs/CONTRIBUTING.md]: @felixcheung Yeah right. Thx for pointing it out!

**Comment by felixcheung** [comment]
LGTM

**Comment by AhyoungRyu** [comment]
Is this good to go now? :)

**Comment by felixcheung** [comment]
Merging if there is no more discussion.

**Pull request closed without merging**